HealthcareApp/src/Healthcare.Application/Schedules/Notifications/UserRegisteredNotificationHandler.cs

```diff
@@ -0,0 +1,25 @@
+using Application.Abstractions;
+using Domain.Constants;
+using MediatR;
+
+namespace Healthcare.Application.Schedules.Notifications;
+
+public class UserRegisteredNotificationHandler : INotificationHandler<UserRegisteredNotification>
+{
+    private readonly IScheduleRepository _repository;
+
+    public UserRegisteredNotificationHandler(IScheduleRepository repository)
+    {
+        _repository = repository;
+    }
+
+    public async Task Handle(UserRegisteredNotification notification, CancellationToken cancellationToken)
+    {
+        switch (notification.Role)
+        {
+            case UserRolesConstants.Doctor:
+                await _repository.CreateDefaultWorkingScheduleAsync(notification.UserId);
+                break;
+        }
+    }
```

Comment:
May be we can refactor it so that it handles the message only for doctors? This switchcase gonna break OCP when new roles are added